.github/workflows/reusable_linting_pure_python.yml

```diff
@@ -35,10 +35,10 @@ jobs:
       - name: Install dependencies
         run: |
           pip install ${{inputs.python_folder}}/[dev]
-      - name: Lint with flake8
+      - name: Lint with ruff
         run: |
           cd ${{inputs.python_folder}}/
-          flake8 ${{inputs.linting_folder}} --count
+          ruff check ${{inputs.linting_folder}} --count
```

Comment:
I tried `ruff` and it doesn't have the `--count` flag. The normal `check` call without flags already outputs the number of errors, so removing `--count` should be enough.